appcelerator-archive/amp
# Refactoring amplifier configuration

## How to test
    $ ampmake build
    $ amp cluster create
    $ docker tag appcelerator/amplifier:local localhost:5000/appcelerator/amplifier:local
    $ docker push localhost:5000/appcelerator/amplifier:local
    $ alias mgr='docker run -it --rm --network=hostnet -v $PWD/stacks:/stacks docker --host=m1'
    $ mgr stack deploy -c /stacks/amplifier.stack.yml amplifier

Make sure amplifier is up and running
